site/en/tutorials/images/classification.ipynb

```diff
@@ -1167,7 +1167,7 @@
         "id": "1Ba2LjtkVrXy"
```

Comment:
<p>How about this?</p><p> </p><p>"Applying dropout will randomly set 20% of the neurons to zero during each training epoch. This helps to avoid overfitting on the training dataset."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/992/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='86'/>

Reply:
Thank you for your kind review :)
I updated the comment as your advice which I think is more natural.